### ISSUE TITLE
Windows does not build

### ISSUE THREAD

**Issue by aklinker1** [anime-skip/web-extension#169]
There appears to be an issue with `unplugin-vue-components` that generates wrong paths for Vue components on Windows, see https://github.com/antfu/unplugin-vue-components/issues/185

If you're on windows, try using git bash or WSL to run the application instead. Both use the standard Linux file paths, so it shouldn't cause a problem there.

**Comment by logiczsniper** [comment]
Hey Aaron, I got the chance to look into this a bit more;
- The issue in unplugin-vue-components suggests that it may be linked to a particular PR, but I do not think this is the case:
  1. Just by reading around source code and to verify,
  2. I rolled back to older versions and got the same error.
- After this, it seemed like the issue would take more time to solve so I decided to pursue a hack so I could just run the web extension for my first time 😂 anyway below is that hack, if you don't want to commit this to master I understand, if you do I can open a PR:

```ts
// vite.config.ts
export default defineConfig({
  ...
  plugins: [
    Components({
      ...
      // THIS *****************
      importPathTransform: path => path.startsWith('C:')
          ? path.replaceAll('\\', '\\\\')
          : path
    })
```
